meta/inspiredminds/contao-sibling-navigation/de.yml

```
@@ -0,0 +1,6 @@
+de:
+    title: Sibling Navigation
+    description: >
+        Stellt eine "Büchernavigation" zur Verfügung, mit der man zwischen Nachrichten und Events vor und zurück wechseln kann. Außerdem ist es möglich an das Ende und den Anfang zu springen.
```

Isn't this called "Buchnavigation"?

```suggestion
        Stellt eine "Buchnavigation" zur Verfügung, mit der man zwischen Nachrichten und Events vor und zurück wechseln kann. Außerdem ist es möglich, an das Ende und den Anfang zu springen.
```